templates/main/group_builds.html.ep

```diff
@@ -1,4 +1,5 @@
-% for my $key (reverse sort keys %$result) {
+% use Sort::Versions;
+% for my $key (reverse sort { versioncmp($a, $b) } keys %$result) {
```

Comment:
I guess because you are using an external module this is a "code smell" and you should provide this sorted list from the backend. But LGTM anyway